pkg/console/subresource/consoleserver/types.go

```diff
@@ -11,13 +11,15 @@ package consoleserver
 
 // Config is the top-level console server cli configuration.
 type Config struct {
-	APIVersion    string `yaml:"apiVersion"`
-	Kind          string `yaml:"kind"`
-	ServingInfo   `yaml:"servingInfo"`
-	ClusterInfo   `yaml:"clusterInfo"`
-	Auth          `yaml:"auth"`
-	Customization `yaml:"customization"`
-	Providers     `yaml:"providers"`
+	APIVersion     string `yaml:"apiVersion"`
+	Kind           string `yaml:"kind"`
+	ServingInfo    `yaml:"servingInfo"`
+	ClusterInfo    `yaml:"clusterInfo"`
+	Auth           `yaml:"auth"`
+	Customization  `yaml:"customization"`
+	Providers      `yaml:"providers"`
+	MonitoringInfo `yaml:"monitoringInfo,omitempty"`
+	LoggingInfo    `yaml:"loggingInfo,omitempty"`
```

Comment:
Created a separate field for each config, since it will be easier to read them at the console server side

Reply:
Sounds good to me.